main.tf

```diff
@@ -155,6 +185,38 @@ resource "aws_db_instance" "db_excluding_name" {
   tags = "${merge(var.tags, map("Name", format("%s-%s", var.environment, var.name)), map("Env", var.environment))}"
 }
 
+# The database instance itself with read replica
+
+resource "aws_db_instance" "db_excluding_name_with_read_replica" {
+  count = "${var.database_name == "" && var.engine_type != "aurora" && var.engine_type != "aurora-mysql" && var.engine_type != "aurora-postgresql" && var.replicate_source_db != "" ? 1 : 0}"
+
+  allocated_storage           = "${var.allocated_storage}"
+  allow_major_version_upgrade = "${var.allow_major_version_upgrade}"
+  auto_minor_version_upgrade  = "${var.auto_minor_version_upgrade}"
+  backup_retention_period     = "${var.backup_retention_period}"
+  backup_window               = "${var.backup_window}"
+  copy_tags_to_snapshot       = "${var.copy_tags_to_snapshot}"
+  engine                      = "${var.engine_type}"
+  engine_version              = "${var.engine_version}"
```

Comment:
Goes with `engine`, doesn't make much sense having this without it. Can be removed